docs/v0.9/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -1,73 +1,150 @@
 # GreptimeDB Cluster
 
-The GreptimeDB cluster can run in the [cluster](/contributor-guide/overview.md) mode to scale horizontally.
+The GreptimeDB cluster can run in cluster mode to scale horizontally. For production usage, we highly recommend deploying it in Kubernetes.
 
-## Install gtctl and run playground
+## Use Docker Compose
```

Comment:
The first sentence just said we recommend deployment on kubernete, but the first section is about docker compose. I think move this behind kubernetes section.

Reply:
+1